.github/workflows/release-published.yml

```diff
@@ -222,4 +222,46 @@ jobs:
         run: mvn -B clean deploy -DskipTests=true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          TARGET_ARTIFACT_REPOSITORY: github
\ No newline at end of file
+          TARGET_ARTIFACT_REPOSITORY: github
+
+  deploy_xsd:
+      name: Upload xsds
+      needs: [ setup ]
+      runs-on: ubuntu-20.04
+      outputs:
+          tag: ${{ steps.collect-data.outputs.tag }}
+          version: ${{ needs.setup.outputs.version }}
+      steps:
+      - name: Download liquibase xsd
+        uses: actions/checkout@v3
+        with:
+          # Relative path under $GITHUB_WORKSPACE to place the repository
+          path: liquibase-core  
+          repository: "liquibase/liquibase"
+    
+      - name: Download liquibase-pro xsd
+        uses: actions/checkout@v3
+        with:
+        # Relative path under $GITHUB_WORKSPACE to place the repository
+          path: liquibase-pro
+          repository: "liquibase/liquibase-pro"
```

Comment:
I don't think the default token has read access to liquibase-pro. You probably need to add `token: ${{ secrets.BOT_TOKEN }} to `with`